Sources/SnapshotTesting/Diffable.swift

```diff
@@ -38,7 +38,10 @@ extension String: Diffable {
   public static func diffableDiff(_ fst: String, _ snd: String) -> (String, [XCTAttachment])? {
     guard fst != snd else { return nil }
 
-    let hunks = chunk(diff: diff(fst.split(separator: "\n"), snd.split(separator: "\n")))
+    let hunks = chunk(diff: diff(
+      fst.split(separator: "\n", omittingEmptySubsequences: false),
```

Comment:
Surprising gotcha with the default behavior! You lose empty lines, even in the middle of a string!

``` swift
let str = """
Hello

And

Goodbye
"""

str.split(separator: "\n")
// ["Hello", "And", "Goodbye"]

str.split(separator: "\n").joined(separator: "\n") == str
// false
```

Reply:
we need tests!